aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -32,7 +32,7 @@ dns = ["tokio/rt"]
 default = ["default-provider", "rustls", "rt-tokio", "dns", "tcp-connector"]
 
 [dependencies]
-aws-sdk-sts = { path = "../../sdk/build/aws-sdk/sdk/sts", optional = true }
+aws-sdk-sts = { path = "../../sdk/build/aws-sdk/sdk/sts", default-features = false, optional = true }
```

Comment:
Should the `rustls` and `native-tls` features in `aws-config` also enable the equivalent features on the STS crate? Or to reframe the question, is it working by coincidence right now without explicitly enabling those?

Reply:
I had that previously and removed it because it didn't seem to matter. The test runs and doesn't detect `rustls` even though `aws-config` is a dependency so I believe we're good. Personally, my gut tells me to include the features but I can't articulate why so I'm ignoring the feeling.

Reply:
@jdisanti they should not: this is because the only purpose of those in the STS crate are to support `from_conf` (constructing the client _without_ a connection). But this crate will always provide a connection when constructing a client.